src/main/scala/org/dbpedia/spotlight/wikistats/JsonPediaParser.scala

```diff
@@ -112,17 +154,44 @@ class JsonPediaParser(inputWikiDump:String, lang:String)
    */
   def getSfURI(): RDD[(Long,String,String)]= {
 
-    val sfUriRDD = dfWikiRDD.select("wid","links.description","links.id")
+    /*
```

Comment:
If this commented code snippet is important then can you add some hints? otherwise removing it would be the best